.github/workflows/release.yml

```diff
@@ -1,94 +0,0 @@
-name: Release
-
-on:
-  push:
-    branches:
-      - "main"
```

Comment:
This workflow was trying to publish every PR merged. This does not allow to group different changes to release together.